packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -268,21 +268,21 @@ const styles = StyleSheet.create({
         color: semanticColor.text.primary,
     },
     labelWithDescription: {
-        paddingBlockEnd: sizing.size_050,
+        paddingBlockEnd: sizing.size_040,
     },
     labelWithNoDescription: {
-        paddingBlockEnd: sizing.size_150,
+        paddingBlockEnd: sizing.size_120,
     },
     description: {
         color: semanticColor.text.secondary,
-        paddingBlockEnd: sizing.size_150,
+        paddingBlockEnd: sizing.size_120,
     },
     errorSection: {
         flexDirection: "row",
         gap: sizing.size_100,
```

Comment:
The gap will also need to be updated to `sizing.size_080` for 8px! ([figma](https://www.figma.com/design/VbVu3h2BpBhH80niq101MHHE/%F0%9F%92%A0-Main-Components?node-id=13502-9799&t=03w7JHtnAShP2vHA-4))

<img width="487" alt="image" src="https://github.com/user-attachments/assets/d77750f0-0949-4093-acd8-3e855be8d2ea" />

Noticed this change in the [Chromatic snapshot](https://www.chromatic.com/test?appId=5e1bf4b385e3fb0020b7073c&id=67ed7611e053bbc80bf9f3f3)! I was expecting there'd be no visual changes since things would be converted to the new sizing!